java-client/barrage-examples/src/main/java/io/deephaven/client/examples/SubscribeExampleBase.java

```diff
@@ -25,6 +26,12 @@
 
         @CommandLine.Option(names = {"-s", "--serial"}, required = true, description = "Serial mode")
         boolean serial;
+
+        @CommandLine.Option(names = {"--tail"}, required = false, description = "Tail viewport size")
+        int tailSize = 0;
+
+        @CommandLine.Option(names = {"--head"}, required = false, description = "Header viewport size")
+        int headerSize = 0;
```

Comment:
We should probably put these size parameters outside of the Mode flags. The mode flags are exclusive with one another. I believe they can simply be moved up a level.